config/locales/views/groups/he.yml

```
@@ -0,0 +1,45 @@
+        add_assignment_btn: "+ הוסף משימה"
+        assignment_table_head:
+          name: "שם"
+          deadline: "תאריך יעד"
+          grading: "דירוג"
+          actions: "פעולות"
+          grade: "ציון"
+        time_remaining_html: "<strong>זמן שנותר:</strong> %{time_remaining}"
+        start_working_btn: "התחל לעבוד"
+        see_work_done: "ראה את העבודה שלך"
+        not_submitted: "לא הוגש!"
+        show_submission: "הצג הגשה"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)